trivial change to make manpage more usefull (and lintian less grumpy)

--- a/doc/cgdb.1
+++ b/doc/cgdb.1
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.41.1.
 .TH CGDB "1" "January 2013" "CGDB 0.6.7" "User Commands"
 .SH NAME
-CGDB \- manual page for CGDB 0.6.7
+CGDB \- curses based frontend to GDB
 .SH DESCRIPTION
 .SS "CGDB Usage:"
 .IP
